ee/modules/650-runtime-audit-engine/openapi/config-values.yaml

```diff
@@ -4,3 +4,137 @@ properties:
     type: boolean
     description: |
       If true, all events will be duplicated to stdout.
+  resourcesRequests:
+    required: ["mode"]
+    type: object
+    description: |
+      Max amounts of CPU and memory resources that the pod can request when selecting a node.
+
+      If the `vertical-pod-autoscaler` module is disabled, then these values become the default ones.
+    default: {}
```

Comment:
if `mode` is required, default shouldn't be set ?

Reply:
it is required for some code

Reply:
I copied it from log-shipper module